javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
Instances of `this.decoder` would need to be replaced with the fully explicit `THREE.DRACOLoader.getDecoder()`, as JS only searches up the prototype chain, but you're correct that the member variable isn't necessary. I'm happy to do either, but this felt a bit cleaner to me.